package exceptions;

public class Calculator 
{
	static int div(int a,int b)
	{
		return a/b;
	}

//	public static void main(String[] args) {
//		System.out.println("main starts");
//		try 
//		{
//			int res=div(10,0);
//			System.out.println(res);
//		} catch (Exception e) 
//		{
//			System.out.println("exception handled");
//			e.printStackTrace();
//		}

	}


